code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -1220,11 +1218,15 @@
 			if(L == user || ishuman(L))
 				continue
 			L.apply_damage(aoe, RED_DAMAGE, null, L.run_armor_check(null, RED_DAMAGE), spread_damage = TRUE)
+			if(firsthit)
```

Comment:
"If it's the first hit, do half damage" It does the opposite

Reply:
after damage is applied